database/sql/V057__update_order_execution.sql

```diff
@@ -0,0 +1,12 @@
+-- solver_fee was added in V049__add_full_fee_amount.sql
+-- Limit order's fee is no longer calculated beforehand, but provided by the solvers at the time of solving. However, under colocation, this fee is not reported to the autopilot but calculated from the onchain calldata.
+
+ALTER TABLE order_execution 
+  DROP COLUMN solver_fee;
```

Comment:
Don't we have to add the data that currently exists in this column in the `surplus_fee` column (that gets renamed to `executed_fee`)?
Otherwise we'd lose data which does not seem right.